### ISSUE TITLE
about finetuning

### ISSUE THREAD

**Issue by strongBrave** [microsoft/TRELLIS#319]
Hi. I have two questions about finetuning the model, hope someone can give some insights.

I would like to fine‑tune the SS model on a hand‑only dataset where I have attached a MANO head, but the dataset does not contain the ground‑truth noise (or velocity) required by the original SS model, since we do not have the 3D scanned hand model. Is it possible to train the model without the diffusion (noise) loss, only with the mano loss, joint loss, etc?
